lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -295,3 +311,25 @@ def crashing_callback(fig, stale):
     canvas = FigureCanvasQTAgg(fig)
     fig.stale = True
     assert called
+
+
+@pytest.mark.backend('Qt5Agg', skip_on_importerror=True)
+def test_form_widget_get_with_datetime_field():
+    if not QtWidgets.QApplication.instance():
+        QtWidgets.QApplication()
+    form = [("Field name", datetime(year=2021, month=3, day=11))]
+    widget = _formlayout.FormWidget(form)
+    widget.setup()
+    values = widget.get()
+    assert(values == [datetime(year=2021, month=3, day=11)])
```

Comment:
`assert` needs no parentheses (it's not a function).

Reply:
True. Thanks for the heads up!